src/components/redemption/Confirming.js

```diff
@@ -5,19 +5,17 @@ import StatusIndicator from '../svgs/StatusIndicator'
 
 class Confirming extends Component {
   handleClickButton = () => {
-    const { txHash } = this.props
+    const { txHash, btcNetwork } = this.props
 
     window.open(
-      `https://blockstream.info/tx/${txHash}`,
+      `https://blockstream.info/${btcNetwork === 'testnet' ? 'testnet/' : ''}tx/${txHash}`,
```

Comment:
Not blocking, but any chance we can make this a link instead of a button with `window.open`?

Reply:
Yes, good idea! Addressed in 66dd8b8